_data/themes/beach.yml

```diff
@@ -0,0 +1,32 @@
+colors:
```

Comment:
I'm curious why you chose a yml file rather than some sort of theme_vars.scss file?

Reply:
My guess is that there are other variables in there like the hero and circle images as well as the reference to the theme CSS file. 

@tbaxter-18f any ideas for streamlining or making this theming easier to work with?

Reply:
Yeah my main goal was to try and keep the theme footprint as small as possible so I combined a few different types of variables into the single .yml file vs having separate config files. I feel like it's a lower barrier for entry too for others that might not be familiar with sass but still wanted to contribute their own color palette.

Reply:
I'm fine with the yaml file and think there are good reasons for it, they just weren't readily apparent to me.